src/app/test/dot-workflow-service.mock.ts

```diff
@@ -59,7 +59,7 @@ export const mockWorkflows = [
 ];
 
 export class DotWorkflowServiceMock {
-    getContentWorkflowActions(inode: string): Observable<DotWorkflowAction[]> {
+    getContentWorkflowActions(): Observable<DotWorkflowAction[]> {
```

Comment:
why do you remove the inode parameter?, the real implementation has it: 

https://github.com/dotCMS/core-web/blob/master/src/app/api/services/dot-workflow/dot-workflow.service.ts#L39

Reply:
the ts lint conf was crying about it since `inode` is not being used in the mock

Reply:
use ```_inode``` and tsc will not cry about it, this is a mock so is ok.